partiql-ast/src/main/java/org/partiql/ast/expr/ExprVarRef.java

```diff
@@ -12,22 +11,19 @@
 import java.util.List;
 
 /**
- * TODO docs, equals, hashcode
+ * Represents a variable reference, e.g. {@code foo.bar} or {@code @foo.bar}.
  */
 @Builder(builderClassName = "Builder")
 @EqualsAndHashCode(callSuper = false)
 public final class ExprVarRef extends Expr {
     @NotNull
-    @Getter
     private final IdentifierChain identifierChain;
 
-    @NotNull
-    @Getter
-    private final Scope scope;
+    private final boolean qualified;
```

Comment:
(self-review) simplification of var ref modeling comes from this prior PR comment -- https://github.com/partiql/partiql-lang-kotlin/pull/1688#discussion_r1896937523. No need to have a separate enum to model the `@T` vs `T` var ref.